exercises/zipper/zipper.exs

```diff
@@ -29,6 +29,9 @@ defmodule BinTree do
 end
 
 defmodule Zipper do
+  alias BinTree, as: BT
+  alias Zipper, as: Z
```

Comment:
A couple things:

1) If we're going to use an alias here we probably shouldn't use single letter alias names. It's not seen very often in Elixir libraries or applications.

2) Since we're in the `Zipper` module we can use `t()` in type specs in this module. Or, alternatively, we could use `__MODULE__.t()`, but the plain `t()` is probably more common in the community.

Reply:
I aliased these since `BT` and `Z` are already used in the typespecs. But I think it probably makes more sense to just use the full `BinTree` and `Zipper`.